src/main/resources/biomart-query.xml

```diff
@@ -0,0 +1 @@
+<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE Query><Query  virtualSchemaName = "default" formatter = "TSV" header = "0" uniqueRows = "0" count = "" datasetConfigVersion = "0.6" ><Dataset name = "BIOMART_SPECIES_NAME_gene_ensembl" interface = "default" ><Attribute name = "ensembl_gene_id" /><Attribute name = "ensembl_transcript_id" /><Attribute name = "ensembl_peptide_id" /><Attribute name = "BIOMART_QUERY_ID" /></Dataset></Query>
```

Comment:
It'd be nice to format the XML onto multiple lines.  I use this online formatter to give a quick output: https://www.freeformatter.com/xml-formatter.html.  I'm sure IntelliJ has plugins for XML formatting, too.

Reply:
Done